public/resources/stylesheets/page/_formVerify.scss

```diff
@@ -0,0 +1,221 @@
+.formUser {
+    height: 50em;
+    width: 100em;
```

Comment:
it is not responsive friendly make statics sizes for elements, use % would in a better responsiviness or useing flex, grid, etc